src/docs/product/accounts/early-adopter.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: "Early Adopter Features"
+sidebar_order: 60
+description: "Learn which features are currently in the early adopter phase."
+---
+
+This page lists the features that you'll have access to when you enable the "Early Adopter" toggle in **Settings > General Settings**:
+
+![The Early Adopter toggle enabled in settings.](early-adopter-toggle.png)
+
+Note that features are sometimes released to early adopters in waves, so you may not see a feature immediately upon enabling the "Early Adopter" setting.
+
+Limitations:
+
+- This list does not include new features that aren't controlled by the "Early Adopter" setting, such as alphas, closed betas, or limited availability features that require manual opt-in.
+- This list is not guaranteed to be 100% up-to-date, but it is monitored and updated frequently.
+
+## Current Early Adopter Features
+
+- [Grouping Breakdown](/product/data-management-settings/event-grouping/grouping-breakdown/)
+- [Issue Reprocessing](/product/issues/reprocessing/)
+- [Span Summary](/product/performance/transaction-summary/#span-summary)
+- Organization Domains - Instead of being a path segment, organization slugs are used as a subdomain for [sentry.io](https://sentry.io). For example, `https://acme.sentry.io/issues/` will replace `https://sentry.io/organizations/acme/issues/`. API URLs are unchanged. 
```

Comment:
```suggestion
- Organization Domains - Instead of being a path segment, organization slugs are used as a subdomain for [sentry.io](https://sentry.io). For example, `https://acme.sentry.io/issues/` replaces `https://sentry.io/organizations/acme/issues/`. API URLs are unchanged. 
```